orca-qos/src/main/kotlin/com/netflix/spinnaker/orca/qos/ExecutionPromoter.kt

```diff
@@ -58,61 +51,44 @@ class DefaultExecutionPromoter(
     }
   }
 
-  override fun onApplicationEvent(event: RemoteStatusChangedEvent) =
-    event.source.let { e ->
-      if (e.status == UP) {
-        log.info("Instance is ${e.status}... ${javaClass.simpleName} starting")
-        discoveryActivated.set(true)
-      } else if (e.previousStatus == UP) {
-        log.info("Instance is ${e.status}... ${javaClass.simpleName} stopping")
-        discoveryActivated.set(false)
-      }
-    }
-
-  @Scheduled(fixedDelayString = "\${pollers.qos.promote-interval-ms:5000}")
-  fun promote() {
-    if (!discoveryActivated.get()) {
-      return
-    }
-
+  @VisibleForTesting
+  public override fun tick() {
```

Comment:
nit: `public` is the default visibility for Kotlin.

Reply:
yea.,.. i wasn't sure why, but the test didn't work without it... maybe cuz it's overriding a protected one?

Reply:
Oh. Hmm... escalating the visibility shouldn't even be possible in that case. 😆